js/build.js

```diff
@@ -1162,7 +1161,7 @@ Fliplet.Widget.instance('chat', function (data) {
           messagesQueue.push(messageData);
 
           // Saves new message in QUEUE
-          Fliplet.Storage.set(QUEUE_MESSAGE_KEY, messagesQueue)
+          Fliplet.App.Storage.set(QUEUE_MESSAGE_KEY, messagesQueue)
```

Comment:
there's a reason for messages to be saved into a specific storage outside of the current app and it's not to slowdown the app storage

Reply:
How can we clear the message queue for a specific app then? App list component needs to clear all data related to an app, and I imagine chat message queues would need to be one of them.

Reply:
To be honest now that I look at this it doesn't seem like the chat in-built caching system but rather something else which @hcarneiro added on top of it. Can you explain how this storage is supposed to work and why it's saving incoming messages?

Reply:
e.g. if this is across all apps, does it mean there's a problem on both FV and portals having apps with chat?

Reply:
@squallstar This Storage use was to make sure messages are "sent" when offline. like on WhatsApp or other messaging apps, we allow you to send a message in offline mode. Once it comes back online there's a bit of code that looks in the Store for message to send.

Reply:
I see, @hcarneiro is it safe to move it to app storage then?

Reply:
So this is to support offline caching of unsent messages?

@squallstar What's your thought on this now?

Reply:
I believe so. As long as all other references of `Fliplet.Storage` are also changed to `Fliplet.App.Storage`.

Reply:
@tonytlwu Yes, exactly that.